spec/fabricators/user_fabricator.rb

```diff
@@ -1,7 +1,8 @@
 Fabricator(:user) do
   name 'burgerbob'
   email 'beefersutherland@example.com'
-  password_digest 'password'
+  password 'password'
```

Comment:
Good call on updating this; I think it makes the tests more applicable to what we're doing.